x/launch/keeper/msg_server_request_add_vested_account_test.go

```diff
@@ -14,13 +13,13 @@ import (
 
 func TestMsgRequestAddVestedAccount(t *testing.T) {
 	var (
-		invalidChain, _            = sample.ChainID(0)
-		coordAddr                  = sample.AccAddress()
-		addr1                      = sample.AccAddress()
-		addr2                      = sample.AccAddress()
-		addr3                      = sample.AccAddress()
-		k, pk, srv, _, sdkCtx, cdc = setupMsgServer(t)
-		ctx                        = sdk.WrapSDKContext(sdkCtx)
+		invalidChain, _          = sample.ChainID(0)
+		coordAddr                = sample.AccAddress()
+		addr1                    = sample.AccAddress()
+		addr2                    = sample.AccAddress()
+		addr3                    = sample.AccAddress()
+		k, pk, srv, _, sdkCtx, _ = setupMsgServer(t)
```

Comment:
Maybe we should remove the `codec.Marshaler` parameter from the `setupMsgServer` method. We will not use it anymore

Reply:
We can actually check for this. I will merge this PR so that we can already resolve potential conflicts